pkgs/development/compilers/bs-platform/default.nix

```diff
@@ -0,0 +1,15 @@
+{ stdenv, fetchgit, ninja, nodejs, python35, ... }:
```

Comment:
does it specifically need python35? otherwise i would recommend python3
```suggestion
{ stdenv, fetchgit, ninja, nodejs, python35, ... }:
```

Reply:
python35 is required. I also don't see any difference in your suggestion, is it possibly outdated?